### DISCUSSION TITLE
WalletConnect BUG

### DISCUSSION THREAD

**Discussion by pausanchezv** [WalletConnect/walletconnect-monorepo#3734] [Developer Support] [answered]
Hello, I'm having trouble with a bug that I will come across with no matter what I do. After connecting the wallet I always get this error in the browser console:

```
web3.d21774d5.js:20 Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://verify.walletconnect.com') does not match the recipient window's origin https://mywebsite.xyz
```

So, basically, despite having added all my origins to my Wallet Connect App I keep getting this. Is there any step I am missing? I added my origins to the option 'Allowed Domains' and my domain has been verified showing a green checkmark.

I went through the docs and It seems that I did all okay. I have also joined the discord channel but I have no way to ask anything there.

Thanks.

**Comment by glitch-txs** [comment, marked as answer]
please update your dependencies, this was fixed on the latest version of WalletConnect/Wagmi
